### PR TITLE
Unify makefiles for: SBUC, AppBroker, Acc-tests and Upgrade tests

### PR DESCRIPTION
**Pull request by piotrmiskiewicz**
**Description**

Changes proposed in this pull request:
- adapt SBUC, Application Broker, acceptance test and upgrade test to generic jobs

**Related issue(s)**
Related: https://github.com/kyma-project/kyma/issues/5742

### CODE REVIEW

File: components/application-broker/README.md [hunk 1]
```diff
@@ -25,7 +25,10 @@ These Go and Dep versions are compliant with the `buildpack` used by Prow. For m
 
 ## Development
 
-Before each commit, use the `before-commit.sh` script or the `make build` command to test your changes.
+Before each commit, use the `make verify` command to test your changes. To build an image, use the `make build-image` command with DOCKER_PUSH_REPOSITORY and DOCKER_PUSH_DIRECTORY envs, for example:
```
Comment (klaudiagrz) [inline components/application-broker/README.md]:
```suggestion
Before each commit, use the `make verify` command to test your changes. To build an image, use the `make build-image` command with **DOCKER_PUSH_REPOSITORY** and **DOCKER_PUSH_DIRECTORY** variables, for example:
```

File: components/service-binding-usage-controller/README.md [hunk 2]
```diff
@@ -41,4 +41,8 @@ Use the following environment variables to configure the application:
 
 ## Development
 
-Use the `before-commit.sh` script or the `make build` command to test your changes before each commit.
+Use the `make verify` command to test your changes before each commit. To build an image, use the `make build-image` command with DOCKER_PUSH_REPOSITORY and DOCKER_PUSH_DIRECTORY envs, for example:
```
Comment (klaudiagrz) [inline components/service-binding-usage-controller/README.md]:
```suggestion
Use the `make verify` command to test your changes before each commit. To build an image, use the `make build-image` command with **DOCKER_PUSH_REPOSITORY** and **DOCKER_PUSH_DIRECTORY** variables, for example:
```

File: tests/acceptance/README.md [hunk 3]
```diff
@@ -7,7 +7,10 @@ The tests are written in Go. Each component or group of scenarios has a separate
 
 ## Usage
 
-To test your changes and build an image, use the `make build build-image` command.
+To test your changes and build an image, use the `make build-image` command with DOCKER_PUSH_REPOSITORY and DOCKER_PUSH_DIRECTORY envs, for example:
```
Comment (klaudiagrz) [inline tests/acceptance/README.md]:
```suggestion
To test your changes and build an image, use the `make build-image` command with **DOCKER_PUSH_REPOSITORY** and **DOCKER_PUSH_DIRECTORY** variables, for example:
```

File: tests/end-to-end/upgrade/README.md [hunk 4]
```diff
@@ -136,7 +136,10 @@ Run the application using Helm:
 
 ### Verify the code
 
-Use the `before-commit.sh` script or the `make build` command to test your changes before each commit.
+Use the `make verify` command to test your changes before each commit. To build an image, use the `make build-image` command with DOCKER_PUSH_REPOSITORY and DOCKER_PUSH_DIRECTORY envs, for example:
```
Comment (klaudiagrz) [inline tests/end-to-end/upgrade/README.md]:
```suggestion
Use the `make verify` command to test your changes before each commit. To build an image, use the `make build-image` command with **DOCKER_PUSH_REPOSITORY** and **DOCKER_PUSH_DIRECTORY** variables, for example:
```

File: tests/acceptance/Makefile [hunk 5]
```diff
@@ -1,43 +1,13 @@
...
-.PHONY: clean
-clean:
-	rm -f gateway.bin
-	rm -f env-tester.bin
-	rm -f client.bin
-
-#.PHONY: path-to-referenced-charts
-#path-to-referenced-charts:
-#	@echo "resources/core"
+.PHONY: path-to-referenced-charts
```
Comment (polskikiel) [inline tests/acceptance/Makefile]: please remove that target from all makefiles, I've discussed it with jasiu

File: tests/end-to-end/upgrade/chart/upgrade/values.yaml [hunk 6]
```diff
@@ -2,8 +2,8 @@ containerRegistry:
   path: eu.gcr.io/kyma-project
 
 image:
-  dir: develop/tests/
-  version: "73838670"
+  dir:
```
Comment (polskikiel) [inline tests/end-to-end/upgrade/chart/upgrade/values.yaml]: shouldn't it be `pr/tests/`??

**Pull request merged**